src/core/ad.jl

```diff
@@ -189,3 +202,7 @@ for F in (:link, :invlink)
         end
     end
 end
+
+## Zygote support ##
+
+Zygote.@nograd Bijectors._debug
```

Comment:
Maybe move this to Bijectors?

Reply:
Hmm sure, I thought Bijectors would then have to depend on Zygote. But I forgot that we can use Requires.

Reply:
Yes, I thought one could just add a file `src/compat/zygote.jl` with the required definitions for Zygote.